participants/SingersMusic.md

```diff
@@ -0,0 +1,87 @@
+# <Project Name>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+
+ ## (0) Registration Submission, Project Name, and Team Slack Handles
+
+*Please provide the Submission ID from your [Slingshot registration confirmation](https://slingshot.filecoin.io/register-now).  Also, confirm your project's name and the Filecoin Slack handles for your teammates.*
+
+Slack is https://app.slack.com/client/T01B4QHHR47/D01C990MFK3
+## (1) Brief Project Description
+
+*Add your project's description here.*
+一款创造属于自己的音乐网页
+Create your own music page
+
+## (2) Link to Project UI
+
+*Please include a link to your project application or UI. It does not need to be super polished or completely functional. We will be reviewing this link periodically to verify that your project is eligible for final rewards.*
+网页链接 :http://singersmusic.gctech.xin/
+Website link :http://singersmusic.gctech.xin/
+
+## (3) What does your application/UI do?
+
+*Give us a brief description of what your application/UI does and how users are intended to interact with it.*
+随着会出现自己常常听的音乐在网上被下架,下载之后会没有地方整理,会非常的乱,导致用户体验贼差.有了这个网页可以创造一个属于自己的音乐网站,记录喜欢的歌手丶歌曲.对此处理相关分类.
+As the music that I often listen to is taken off the Internet, there will be no place to organize it after downloading, which will be very messy, resulting in poor user experience. With this webpage, you can create your own music website and record favorite singers. Dian Songs. Processing related classifications.
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*You can reference the [list of curated datasets in this repo](https://github.com/filecoin-project/slingshot/blob/master/datasets.md). If you'd like to use a dataset that is not on that list, please nominate your chosen dataset by updating the curated datasets table in this PR. If nominating a dataset, do you own or have exclusive rights to the data you plan to use?*
+暂且无使用官网数据,数据来源用户上传.
+There is no official website data for the time being, and the data source is uploaded by the user.
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*What sorts of data will you collect? Do you own or have exclusive rights to the data you plan to use? Is the data license available online? Does the data contain any sensitive or illegal information?*
+
+数据来源于用户授权上传存储。
+
+Data from the user authorization upload store.
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+*Give us as realistic an estimate as you possibly can at this stage. We understand it's impossible to have perfect information before really starting your project!*
+
+We have probably 5P data planning store to the Filecoin Network
```

Comment:
This seems high for a music collection.   How many users would provide music for a 5P size?